fwutil/lib.py

```diff
@@ -729,6 +972,7 @@ class ComponentStatusProvider(PlatformDataProvider):
     ComponentStatusProvider
     """
     HEADER = [ "Chassis", "Module", "Component", "Version", "Description" ]
+    AU_STATUS_HEADER = [ "Component", "Status", "Info", "Boot" ]
```

Comment:
Please include `Version` field in `fwutil show update status` command output.

Snippet from HLD:
```
root@sonic:~# fwutil show update status
Firmware auto-update performed for cold reboot
Component      Version    Status     Info
-------------  ---------  ---------  --------------------------------------
MSN2700/SSD    4/5        scheduled  installation scheduled for cold reboot
MSN2700/CPLD1  5/10       installed  need cold reboot to be completed
```